src/Components/Homepage.jsx

```diff
@@ -1,13 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
 import Section from './home/Section';
 import Calendar from './home/Calendar';
 
 export default function Homepage() {
+  const [text, setText] = useState('Loading');
+  useEffect(() => {
+    axios.get(`${process.env.REACT_APP_API_URL}/announcements`)
```

Comment:
I don't like using process.env in the component like this.  I'd rather see it pulled out into it's own file/class somehow.  I'm not sure exactly but I'm gonna do some googling to see if I can find examples.  Just looks goofy using it directly like that

Reply:
Okay so I think we should have a file that creates our apiClient.  Something like:
`import axios from "axios";

const apiClient = axios.create({
  baseURL: process.env.baseURL
});

export default apiClient;`

Then every request will be using the same client.  In the future when we add authentication and authorization to our services calls, we'll want one single place where we can add those. Components can import the client and simply call apiClient.get('calendar').  Does that make sense? Do you see what I'm getting at?

Reply:
For anyone else reading this here's what I was saying... https://alligator.io/react/axios-react/#base-instance

Reply:
I made changes based on the article above.  Will make most API calls much cleaner, though the calendar call is a little messier.